packages/client/src/types.d.ts

```diff
@@ -0,0 +1,12 @@
+import { Metadata } from '@appland/models';
+
+export type FindingStatusListItem = {
+  app_id: string;
+  identity_hash: string;
+  status: string;
+};
+
+export type AppMapListItem = {
+  scenario_uuid: string;
+  metadata: Metadata;
+};
```

Comment:
Type declaration files are meant to describe JavaScript types, so we shouldn't need one to describe types in this package. Can these types find better homes in another location? I'd recommend they be placed with the API that returns them, but I'm fine with anything you can come up with.